test/e2e/service.go

```diff
@@ -95,6 +102,130 @@ var _ = ginkgo.Describe("Services", func() {
 		framework.ExpectNoError(err)
 	})
 
+	// The below series of tests queries nodePort services with hostNetwork:true and hostNetwork:false pods as endpoints.
+	// It queries the service both with a small (10 bytes) and a large (10000 bytes) payload. The payload is added to
```

Comment:
I think we need a little bit more coverage here. This test uses 10 bytes and 10000 bytes and only TCP. With TCP the negotiated MSS will be used so there is no fragmentation in this case, only segmentation. Meaning that there will never be fragmentation at any point in this test, which is something we should cover.

I think we need to also include UDP here to cover fragmentation. Additionally with sending a 10000 byte packet, the sender side will fragment this. In other words node Z will fragment the packet on its way out. 
However, we also want to test that the fragmentation behavior of OVN works as expected. With OVN when we set the gateway mtu to be 1400, it should act like its external interface has a 1400 byte MTU. Therefore we want to send a packet from node z to node y that is not fragmented, but is too large for OVN, which will force OVN to send ICMP frag needed back to the node z (note this case may also work for TCP with a host network endpoint on node z). What you can do is this (as a reply to a service request):

1. node z host network pod responds with a 1420 byte packet
2. node y receives the packet, OVN thinks MTU is too large, sends ICMP frag needed back to node z
3. node z sends 2 fragments towards y
4. node x receives the fragments and reassembles

Reply:
I tried tons of different ways and now I settled with agnhost's echo service; that seems to work and I think it covers what needs to be covered. wdyt